src/content/docs/apm/agents/java-agent/installation/java-install-overview.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: An overview of the Java APM installation process
+tags:
+  - Agents
+  - Java agent
+  - Installation
+translate:
+  - jp
+metaDescription: An overview of how to download and install New Relic's APM agent for Java.
+---
+
+Our Java APM agent auto-instruments your code so you can start monitoring applications. Read this for an overview of the generic APM installation process. For more specific instructions, use [monitor your Java app](/docs/apm/agents/java-agent/getting-started/monitor-your-java-app) to find the specific resource for your Java app configuration.
+
+## 1. Prepare to install [#overview]
+
+Check the following:
+
+* Review the [compatibility and requirements](/docs/agents/java-agent/getting-started/compatibility-requirements-java-agent).
+* Check if your environment requires [additional or alternative install steps](/docs/agents/java-agent/additional-installation).
+
+When you're ready, use the guided installation for an automated install. If you're in the EU, select that option. Or for a manual approach, follow the instructions in this document to complete a basic Java agent installation. Either way, you need a New Relic account. ([It's free, forever](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#how-pricing-works).)
```

Comment:
The link to the pricing page seems unnecessary to me.

Reply:
I changed the link to the signup page instead.